applications/search/src/pages/api-details-page/sticky-menu/sticky-menu.component.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
dette ser ut som et generell komponent, kan det ligge i components?

Reply:
moved to components